### PR TITLE
bumped seekret sniffer to rolling version 2 

### PR DESCRIPTION
**Pull request by val06**

### CODE REVIEW

File: aws-traffic-mirroring/templates/vpc-mirroring-target-instance.yaml [hunk 1]
```diff
@@ -19,7 +19,7 @@ Parameters:
   SnifferVersion:
     Type: String
     Description: Seekret Sniffer Version
-    Default: 1
+    Default: 2
```
Comment (guyarb) [inline aws-traffic-mirroring/templates/vpc-mirroring-target-instance.yaml:22]: you need to update the configuration and the command line

**Review by guyarb — CHANGES_REQUESTED** [review]

File: aws-traffic-mirroring/templates/vpc-mirroring-target-instance.yaml (hunk 1 above)
Comment (val06) [reply aws-traffic-mirroring/templates/vpc-mirroring-target-instance.yaml:22]: which cfg?

**Review by guyarb — DISMISSED** [review]

File: aws-traffic-mirroring/templates/vpc-mirroring-target-instance.yaml (hunk 1 above)
Comment (val06) [reply aws-traffic-mirroring/templates/vpc-mirroring-target-instance.yaml:22]: fixed

File: aws-traffic-mirroring/templates/vpc-mirroring.yaml [hunk 2]
```diff
@@ -25,17 +25,29 @@ Resources:
...
+#    Type: AWS::EC2::TrafficMirrorSession
+#    Properties:
+#      Description: Mirror traffic from !Select [1, !Ref SourceEniList ] to ${TargetEni}
+#      NetworkInterfaceId: !Select [1, !Ref SourceEniList ] # get the relevant element by index (zero-based) from SourceEniList
+#      SessionNumber: 2 # this should be unique for each MirrorSession object
+#      Tags:
+#        - { Key: Name, Value: !Sub '${AWS::StackName}-session' }
+#      TrafficMirrorFilterId: !Ref MirrorAlbTargetTraffic
+#      TrafficMirrorTargetId: !Ref MirrorTarget
+#      VirtualNetworkId: 12346 # this should be unique for each MirrorSession object
```
Comment (guyarb) [inline aws-traffic-mirroring/templates/vpc-mirroring.yaml:50]: why keeping it?

File: aws-traffic-mirroring/templates/vpc-mirroring.yaml [hunk 3]
```diff
@@ -25,17 +25,29 @@ Resources:
...
-      SessionNumber: 1
+      Description: Mirror traffic from !Select [0, !Ref SourceEniList ] to ${TargetEni}
+      NetworkInterfaceId: !Select [0, !Ref SourceEniList ] # get the relevant element by index (zero-based) from SourceEniList
+      SessionNumber: 1 # this should be unique for each MirrorSession object
       Tags:
         - { Key: Name, Value: !Sub '${AWS::StackName}-session' }
       TrafficMirrorFilterId: !Ref MirrorAlbTargetTraffic
       TrafficMirrorTargetId: !Ref MirrorTarget
-      VirtualNetworkId: 12345
+      VirtualNetworkId: 12345 # this should be unique for each MirrorSession object
```
Comment (guyarb) [inline aws-traffic-mirroring/templates/vpc-mirroring.yaml:38]: parameterized it?

**Review by guyarb — CHANGES_REQUESTED** [review]

File: aws-traffic-mirroring/templates/vpc-mirroring.yaml (hunk 2 above)
Comment (val06) [reply aws-traffic-mirroring/templates/vpc-mirroring.yaml:50]: to explain the changes that need to be done to add additional mirror sessions (easier to explain than writing a lot of words in Readme)

File: aws-traffic-mirroring/templates/vpc-mirroring.yaml (hunk 3 above)
Comment (val06) [reply aws-traffic-mirroring/templates/vpc-mirroring.yaml:38]: what do you mean? I can't dynamically calculate it

File: aws-traffic-mirroring/templates/vpc-mirroring-target-instance.yaml [hunk 4]
```diff
@@ -95,7 +95,6 @@ Resources:
                   STORAGE_PROVIDER=${BucketProvider}
                   BUCKET_NAME=${BucketName}
                   FOLDER_NAME=${BucketFolderName}
-                  BPF_FILTER=udp and port 4789
```
Comment (guyarb) [inline aws-traffic-mirroring/templates/vpc-mirroring-target-instance.yaml:98]: make sure the sniffer does not have "default bpf filter"

**Review by guyarb — CHANGES_REQUESTED** [review]

File: aws-traffic-mirroring/templates/vpc-mirroring-target-instance.yaml (hunk 4 above)
Comment (val06) [reply aws-traffic-mirroring/templates/vpc-mirroring-target-instance.yaml:98]: it doesn't have

**Review by guyarb — APPROVED** [review]

**Pull request merged**